spec/classes/puppet_agent_spec.rb

```diff
@@ -104,7 +104,7 @@
 
         it { is_expected.to contain_class('puppet::agent::service::systemd').with_enabled(false) }
         case os
-        when /\Adebian-/, /\A(redhat|centos|scientific)-7/, /\Afedora-/, /\Aubuntu-(16|18)/, /\Aarchlinux-/
+        when /\Adebian-/, /\A(redhat|centos|scientific)-7/, /\Afedora-/, /\Aubuntu-(16|18|20)/, /\Aarchlinux-/
```

Comment:
This is all the Ubuntu versions we support. I won't repeat it for every line in this file
```suggestion
        when /\Adebian-/, /\A(redhat|centos|scientific)-7/, /\Afedora-/, /\Aubuntu-/, /\Aarchlinux-/
```